geode-for-redis/src/main/java/org/apache/geode/redis/internal/commands/RedisCommandType.java

```diff
@@ -394,22 +475,28 @@
   private final Parameter parameterRequirements;
   private final Parameter deferredParameterRequirements;
   private final RedisCommandSupportLevel supportLevel;
+  private final Category category;
 
-  RedisCommandType(CommandExecutor commandExecutor, RedisCommandSupportLevel supportLevel) {
-    this(commandExecutor, supportLevel, new Parameter().custom(c -> {
+  RedisCommandType(CommandExecutor commandExecutor, Category category,
+      RedisCommandSupportLevel supportLevel) {
+    this(commandExecutor, category, supportLevel, new Parameter().custom(c -> {
```

Comment:
do we need to call "custom" to add a noop Consumer lamda? It seems like `new Parameter()` is all we need.